platform_api/orchestrator/kube_orchestrator.py

```diff
@@ -904,7 +911,12 @@ async def get_scheduled_jobs(self, jobs: list[Job]) -> list[Job]:
         return scheduled
 
     async def get_schedulable_jobs(self, jobs: list[Job]) -> list[Job]:
-        job_pods = [self._create_pod_descriptor(job) for job in jobs]
+        job_pods = []
+        for job in jobs:
+            try:
+                job_pods.append(self._create_pod_descriptor(job))
+            except JobError as exc:
+                logger.debug("Job %r cannot be scheduled. Reason: %s", job.id, exc)
```

Comment:
If we just ignore such jobs here, they will be stuck in a PENDING state.

Reply:
They will be moved to FAILED state in [suspend_jobs](https://github.com/neuro-inc/platform-api/pull/1978/files#diff-4d5de340f902c70dd22473d166d6b7d35c4a17b5788d4fa53055ca0b51259160R454) method. Scheduler will always try to preempt space for unschedulable job and will fail job if it doesn't fit into cluster.

Reply:
This [test](https://github.com/neuro-inc/platform-api/pull/1978/files#diff-01cb90572193c021c6325aca555011a27e7b1c371d76033f4dcb09219e002c4aR2980) should cover the case